### ISSUE TITLE
AM2 Etherium Power being float makes 0 sense.

### ISSUE THREAD

**Issue by AirBurn0** [RubilaxXxx/ArsMagica2-5#3]
Not sure why, but it's float type.
Nothing in AM2 really uses less than 1 (or 0.01) Etherium, so why should it be float?
For now it's okay, but if I do some AM2&AE2 integration, this can cause some rounding errors resulting infinity etherium sources.
Well, I just hate it being float. Even in Thaumcraft it's just int (divided by 100 for display).
That probably isn't wont worth changing.

**Comment by AirBurn0** [comment]
> i mean if you look at ae2 power system its using double not int same for meka and probly other mods.

yup but CoFH core treat RF basically like int, maybe GregTech also uses int/long (not really sure).
Exact amount of energy in AE2 is generally not that important and can usually be gathered in a wide range of numbers, just like in Mekanism, but that's not really deal for AM2, since it's more arcane-magic-shit mod that totally not centred about energy production/consumption, thus float feels really strange choice but, again, that probably wont worth changing.

**Comment by RubilaxXxx** [comment]
I leave this here, i'm rewriting power stuff (hopefully making it more stable), nothing to release yet +busy irl so it gonna drop probably next month, i ended up going for int   instead of float :)

**Comment by RubilaxXxx** [comment]
Its just a rewrite, ingame won't change much than how it was before other than crystal wrench & magitech staff
